### PR TITLE
Committee logo added

### PR DESCRIPTION
**Pull request by borrelcie**
The logo's are dumped at https://borrelcie.vodka/tmp/ because i didn't have accesses to upload pictures :)

### CODE REVIEW

File: src/Application/Committees/CommitteesRepository.php [hunk 1]
```diff
@@ -15,140 +15,159 @@ public function __construct()
...
+                "logo" => "https://borrelcie.vodka/tmp/Fotocie.png",
                 "link" => "fotocie",
                 "page" => "pages.association.committees.fotocie",
             ],
             [
                 "id" => 9,
                 "title" => "Fraccie",
-                "email" => "",
+                "email" => "Fraccie@professorfrancken.nl",
+                "logo" => "https://borrelcie.vodka/tmp/Fraccie.png"
```
Comment (MarkRedeman) [inline src/Application/Committees/CommitteesRepository.php]: You need a comma `,` here.

File: src/Application/Committees/CommitteesRepository.php [hunk 2]
```diff
@@ -15,140 +15,159 @@ public function __construct()
...
                 "link" => "fraccie",
                 "page" => "pages.association.committees.fraccie",
             ],
             [
                 "id" => 10,
                 "title" => "Francken Vrij",
                 "email" => "",
+                "logo" => "https://borrelcie.vodka/tmp/Francken_Vrij.png"
```
Comment (MarkRedeman) [inline src/Application/Committees/CommitteesRepository.php]: You need a comma `,` here.

File: src/Application/Committees/CommitteesRepository.php [hunk 3]
```diff
@@ -15,140 +15,159 @@ public function __construct()
...
                 "link" => "francken-vrij",
                 "page" => "pages.association.committees.francken-vrij",
             ],
             [
                 "id" => 32,
                 "title" => "Intercie",
                 "email" => "",
+                "logo" => "https://borrelcie.vodka/tmp/Intercie.png"
```
Comment (MarkRedeman) [inline src/Application/Committees/CommitteesRepository.php]: You need a comma `,` here.

File: src/Application/Committees/CommitteesRepository.php [hunk 4]
```diff
@@ -15,140 +15,159 @@ public function __construct()
...
                 "link" => "intercie",
                 "page" => null,
             ],
             [
                 "id" => 12,
                 "title" => "Kascie",
                 "email" => "",
+                "logo" => "https://borrelcie.vodka/tmp/Kascie.png"
```
Comment (MarkRedeman) [inline src/Application/Committees/CommitteesRepository.php]: You need a comma `,` here.

File: src/Application/Committees/CommitteesRepository.php [hunk 5]
```diff
@@ -15,140 +15,159 @@ public function __construct()
...
                 "link" => "kascie",
                 "page" => "pages.association.committees.kascie",
             ],
             [
                 "id" => 18,
                 "title" => "Oefensescie",
                 "email" => "",
+                "logo" => "https://borrelcie.vodka/tmp/Oefensescie.png"
```
Comment (MarkRedeman) [inline src/Application/Committees/CommitteesRepository.php]: You need a comma `,` here.

File: src/Application/Committees/CommitteesRepository.php [hunk 6]
```diff
@@ -15,140 +15,159 @@ public function __construct()
...
                 "link" => "oefensescie",
                 "page" => "pages.association.committees.oefensescie",
             ],
             [
                 "id" => 5,
                 "title" => "Representacie",
                 "email" => "",
+                "logo" => "https://borrelcie.vodka/tmp/Representacie.png"
```
Comment (MarkRedeman) [inline src/Application/Committees/CommitteesRepository.php]: You need a comma `,` here.

File: src/Application/Committees/CommitteesRepository.php [hunk 7]
```diff
@@ -15,140 +15,159 @@ public function __construct()
...
                 "link" => "representacie",
                 "page" => "pages.association.committees.representacie",
             ],
             [
                 "id" => 24,
                 "title" => "Sjaarscie",
                 "email" => "",
+                "logo" => "https://borrelcie.vodka/tmp/Sjaarscie.png"
```
Comment (MarkRedeman) [inline src/Application/Committees/CommitteesRepository.php]: You need a comma `,` here.

File: src/Application/Committees/CommitteesRepository.php [hunk 8]
```diff
@@ -15,140 +15,159 @@ public function __construct()
...
                 "link" => "sjaarscie",
                 "page" => "pages.association.committees.sjaarcie",
             ],
             [
                 "id" => 29,
                 "title" => "Sportcie",
                 "email" => "",
+                "logo" => "https://borrelcie.vodka/tmp/Sportcie.png"
```
Comment (MarkRedeman) [inline src/Application/Committees/CommitteesRepository.php]: You need a comma `,` here.

File: src/Application/Committees/CommitteesRepository.php [hunk 9]
```diff
@@ -15,140 +15,159 @@ public function __construct()
...
                 "link" => "sportcie",
                 "page" => null,
             ],
             [
                 "id" => 22,
                 "title" => "Sympcie",
                 "email" => "",
+                "logo" => "https://borrelcie.vodka/tmp/Sympcie.jpg"
```
Comment (MarkRedeman) [inline src/Application/Committees/CommitteesRepository.php]: You need a comma `,` here.

File: src/Application/Committees/CommitteesRepository.php [hunk 10]
```diff
@@ -15,140 +15,159 @@ public function __construct()
...
                 "link" => "sympcie",
                 "page" => "pages.association.committees.sympcie",
             ],
             [
                 "id" => 28,
                 "title" => "Takcie",
                 "email" => "",
+                "logo" => "https://borrelcie.vodka/tmp/Takcie.png"
```
Comment (MarkRedeman) [inline src/Application/Committees/CommitteesRepository.php]: You need a comma `,` here.

File: src/Application/Committees/CommitteesRepository.php [hunk 11]
```diff
@@ -15,140 +15,159 @@ public function __construct()
...
                 "link" => "takcie",
                 "page" => null,
             ],
             [
                 "id" => 27,
                 "title" => "Wiecksie",
                 "email" => "",
+                "logo" => "https://borrelcie.vodka/tmp/Wiecksie.png"
```
Comment (MarkRedeman) [inline src/Application/Committees/CommitteesRepository.php]: You need a comma `,` here.

File: src/Application/Committees/CommitteesRepository.php [hunk 12]
```diff
@@ -15,140 +15,159 @@ public function __construct()
...
                 "link" => "wiecksie",
                 "page" => "pages.association.committees.wiecksie",
             ],
             [
                 "id" => 4,
                 "title" => "s[ck]rip(t|t?c)ie",
                 "email" => "scriptcie@professorfrancken.nl",
+                "logo" => "https://borrelcie.vodka/tmp/Scriptcie.png"
```
Comment (MarkRedeman) [inline src/Application/Committees/CommitteesRepository.php]: You need a comma `,` here.

**Comment by ProtoPrism** [comment]
Some slight errors, added this pull to beta.professorfrancken.nl

**Comment by Sbte** [comment]
Committed by acc293d

**Pull request closed without merging**